test/IntegrationTests.jl

```diff
@@ -0,0 +1,30 @@
+using DataDeps
+using Base.Test
+
+ENV["DATADEPS_ALWAY_ACCEPT"]=true
+
+@testset "Testing DataDyrad DataDeps" begin
+    include("references/DataDryad Drought.txt")
+    include("references/DataDryad Plasticity.txt")
```

Comment:
Testing the copy that is in `"references" is not ideal.
Because if one makes a change, but forgets to update the reference copy,
then the change won't be reflected in the results.

I'm not sure the best way to avoid it.
One option is `eval(parse(generate(....)`.
(wrap that into a function)